test/commands/commerce/store/epn.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Use a similar directory structure and name for tests. This test must be test/commands/commerce/extension/points/list.tests.ts